README.md

```diff
@@ -50,7 +55,9 @@ See [source](examples/shaders/sky-shader/src/lib.rs) for full details.
 
 ## Our Vision & Community Contributions
 
-`rust-gpu` is a project that we at Embark think has the potential to change the way GPU programming works in multiple ways. One of the primary things we think it can change is opening the door to leverage the open source culture of sharing and improving each others' code, and our end goal and vision for `rust-gpu` is to develop it very much in tandem with the community. However, the project is still in quite early stages and has a very small team, so in order to be productive and guide the project to where we ultimately want it to go, as of right now, we need to focus on the use-cases which we have on our own projects and which we personally believe in. What this means practically is that it is unlikely that we'll be able to accept major changes from community members at this time. If you have a large change you would like to make, please file an issue and/or ask on our Discord in the #rust-gpu channel to see if it is something we'll be able to accept *before* working on it, as it really stinks to have to turn down stuff that community members have poured their time and effort into. As the project matures, we'll in theory be able to accept more input from the community and move closer and closer to the goals outlined above. Thank you so much for your understanding!
+`rust-gpu` is a project that we at Embark think has the potential to change the way GPU programming works in multiple ways. One of the primary things we think it can change is opening the door to leverage the open source culture of sharing and improving each others' code, and our end goal and vision for `rust-gpu` is to develop it very much in tandem with the community. However, the project is still in quite early stages and has a very small team working on it, so in order to be productive and guide the project to where we ultimately want it to go, as of right now, we need to focus on the our own primary use cases for our projects at Embark. 
```

Comment:
"need to focus on ~~the~~ our own primary use cases"

Reply:
thanks!